installer/MSIX/appxmanifest.xml

```diff
@@ -48,7 +51,44 @@
             </desktop5:ItemType>
           </desktop4:FileExplorerContextMenus>
         </desktop4:Extension>
+        <uap:Extension Category="windows.fileTypeAssociation" desktop2:AllowSilentDefaultTakeOver="true">
```

Comment:
Getting an error on this line and 62 that the AllowSilentDefaultTakeOver attribute doesn't exist on the uap:Extension element.

Reply:
Created a new [PR ](https://github.com/microsoft/PowerToys/pull/1339)for the changes. There was some issues with merging back the dev/previewPane. Yes it should be[ under `uap3:FileTypeAssocuiation` ](https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-uap-filetypeassociation#attributes)